client/reader/site-subscriptions-manager/reader-site-subscriptions.tsx

```diff
@@ -46,53 +47,44 @@ const ReaderSiteSubscriptions = () => {
 	const recordSearchPerformed = useRecordSearchPerformed();
 	const recordSearchByUrlPerformed = useRecordSearchByUrlPerformed();
 
-	const [ debouncedSearchTerm ] = useDebounce( searchTerm, 600 );
-
-	// Takes the ?s= url query search term and set it to the subscriptions query.
-	useEffect( () => {
-		if ( initialUrlQuerySearchTerm ) {
-			setSearchTerm( initialUrlQuerySearchTerm as string );
-		}
-		// This should only run once
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [] );
-
 	// Update url query when search term changes
 	useEffect( () => {
-		setUrlQuery( SEARCH_KEY, debouncedSearchTerm );
-	}, [ debouncedSearchTerm ] );
```

Comment:
Write the current value of `searchTerm` into the URL query params. The fact that `debouncedSearchTerm` was used was the cause of the bug where the query params disappeared for a brief moment.